src/components/NewTypography/Text.tsx

```diff
@@ -49,6 +55,53 @@ const Text: React.FC<TextProps> = styled.p<TextProps>`
   ${typography}
   ${layout}
   ${({ textTransform }) => textTransform && { 'text-transform': textTransform }}
+
+  ${({ $truncate, $maxCharLen, $value }) => {
+    if (!$truncate || !$value) return;
+    const charLength = $value && $value.length;
+
+    if (charLength >= $maxCharLen / 3) {
```

Comment:
I might need an explainer for this and possibly and added comment to save it in the code as well

Reply:
It was for handling the overflow for account names and notes.
Implementation was incomplete so I removed.
Will be tackled in separate PR.